### PR TITLE
Add Node jar link to other supported platforms

### PR DESCRIPTION
**Pull request by mboussaa**
Screenshot 2024-09-26 at 21 00 29

### CODE REVIEW

**Review by AlexFalessi — APPROVED** [review]

**Comment by fviale** [comment]
there might be a confusion regarding AMD 64, as this is equivalent to x64 or x86_64 used by most today's computer.

Did you change the default node.jar so libraries for this architecture ? Or did you only remove some libraries to make it smaller ?

**Comment by mboussaa** [comment]
I only removed some libraries to make it smaller
@fviale @ankicabarisic

**Comment by ankicabarisic** [comment]
@mboussaa did you modify the name of the .so libraries to the original ones? the test fails with a different name for AMD one

**Comment by mboussaa** [comment]
I will do that in the Release PR, not this one. This is PR has just the front-end links to the new node jars 
@ankicabarisic

**Review by ankicabarisic — APPROVED** [review]

**Comment by mboussaa** [comment]
@fviale what do you suggest ?

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/AddNodeWindow.java [hunk 1]
```diff
@@ -66,8 +66,14 @@ private void build() {
                      "<li><a target='_blank' href='" + Config.get().getRestPublicUrlOrGuessRestUrl() +
                      "/node.jar'>Download</a> the node's JAR file</li>" +
                      "<li>Create credential file (Portal->Create Credentials) </li>" +
-                     "<li>Run from command line:<br/><br/><input type='text' value='java -jar node.jar -f <CREDENTIAL_FILE> -r " +
-                     RMConfig.get().getRMUrl() + "' style='width:270px;border-style:none' disabled></li>";
+                     "<li>Run from command line:<br/><input type='text' value='java -jar node.jar -f <CREDENTIAL_FILE> -r " +
+                     RMConfig.get().getRMUrl() + "' style='width:270px;border-style:none' disabled></li>" +
+                     "<br/><li>Other supported platforms: <a target='_blank' href='" +
+                     Config.get().getRestPublicUrlOrGuessRestUrl() +
+                     "/node-amd-64.jar'>AMD 64</a>, <a target='_blank' href='" +
```
Comment (fviale) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/AddNodeWindow.java:73]: AMD 64 (smaller)

Comment (mboussaa) [reply rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/AddNodeWindow.java]: done

**Review by fviale — APPROVED** [review]

**Pull request merged**